genesyscloud/outbound/resource_genesyscloud_outbound_init_test.go

```diff
@@ -38,7 +39,7 @@ func (r *registerTestInstance) registerTestResources() {
 	providerResources["genesyscloud_outbound_sequence"] = outboundSequence.ResourceOutboundSequence()
 	providerResources["genesyscloud_outbound_settings"] = ResourceOutboundSettings()
 
-	providerResources["genesyscloud_outbound_dnclist"] = ResourceOutboundDncList()
+	providerResources["genesyscloud_outbound_dnclist"] = outbound_dnclist.ResourceOutboundDncList()
```

Comment:
You don't need to update this line, dnc list isn't used by the other resources in th outbound package so you can remove it